tests/all/js_globals/Number.rs

```diff
@@ -29,6 +29,46 @@ fn new() {
         .test()
 }
 
+#[test]
+fn parse_int_float() {
+    project()
+        .file(
+            "src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js::Number;
+
+            #[wasm_bindgen]
+            pub fn parse_int(text: &str, radix: u8) -> Number {
+                Number::parse_int(text, radix)
+            }
+
+            #[wasm_bindgen]
+            pub fn parse_float(text: &str) -> Number {
+                Number::parse_float(text)
+            }
+        "#,
+        )
+        .file(
+            "test.ts", r#"
```

Comment:
⚠️ This needs to be `test.js` to become green on travis before we've addressed [this](https://github.com/rustwasm/wasm-bindgen/blob/efa4a2b8fa5da3e5aecd1a71f060c1954fc58cf4/tests/all/project_builder.rs#L233).

Reply:
I think it's safe to change this to `test.js` now

Reply:
done.